### DISCUSSION TITLE
How to import an existing Applications to ArgoCD

### DISCUSSION THREAD

**Discussion by ckmishra8** [argoproj/argo-cd#16031] [Q&A]
ArgoCD version: 5.23.4

I have few Applications created using a different ArgoCD setup now due to some reason, I had to delete the ArgoCD namespace and create a new ArgoCD setup. But Applications created using older setup are still there and I want to manage it using new ArgoCD setup. Applications created using older setup are still there on the cluster. How can we enforce new ArgoCD setup to manage older Applications rather than creating them again from scratch.

**Comment by pasha-codefresh** [comment]
Hey, @thisischandanmishra, I think you can either install ArgoCD in the same namespace or use this feature (https://argo-cd.readthedocs.io/en/stable/operator-manual/app-any-namespace/) to define the namespaces you want to manage. Also, I believe it's a good approach to store application YAMLs in a Git repository. This tool (https://argocd-autopilot.readthedocs.io/en/stable/) can assist you with that.
